app/src/main/java/org/wikipedia/editactionfeed/AddTitleDescriptionsItemFragment.kt

```diff
@@ -132,10 +139,13 @@ class AddTitleDescriptionsItemFragment : Fragment() {
         viewArticleTitle.text = summary!!.normalizedTitle
 
         if (parent().source == InvokeSource.EDIT_FEED_TRANSLATE_TITLE_DESC) {
-            val spannableDescription = SpannableString(sourceDescription)
-            spannableDescription.setSpan(BackgroundColorSpan(ResourceUtil.getThemedColor(requireContext(), R.attr.text_highlight_color)), 0, sourceDescription.length, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE)
-            viewArticleSubtitle.text = TextUtils.concat(String.format(getString(R.string.translation_source_description), app.language().getAppLanguageCanonicalName(parent().langFromCode)), spannableDescription)
+            viewArticleSubtitleContainer.visibility = VISIBLE
+            viewArticleSubtitleAddedBy.visibility = GONE
+            viewArticleSubtitleEdit.visibility = GONE
+            viewArticleSubtitle.text = sourceDescription
+            callToActionText.text = String.format(getString(R.string.add_translation), app.language().getAppLanguageLocalizedName(parent().langToCode))
```

Comment:
Should it use `getAppLanguageCanonicalName` instead?

Reply:
Done. Good catch!